f2/src/utils/formik/validationSchema.js

```diff
@@ -0,0 +1,27 @@
+const Yup = require('yup')
+
+const phoneRegExp = /^(\+?\d{0,4})?\s?-?\s?(\(?\d{3}\)?)\s?-?\s?(\(?\d{3}\)?)\s?-?\s?(\(?\d{4}\)?)?$/
+const postalCodeRegex = /^([A-Za-z]\d[A-Za-z][-]?\d[A-Za-z]\d)/
+
+//Validation object, defines rules used to validation form data
+const validationSchema = Yup.object().shape({
```

Comment:
I think it's a better idea to co-locate validation schema with different forms. To avoid this js file becoming the contention point

Reply:
Good point, I will move schema definition to the form. However, this validation logic is only for demonstration purpose since there is no existing validation rules for this page.

Reply:
Yup validation logic is now defined within the form. It works as expected.